### ISSUE TITLE
Manually override slug in single file

### ISSUE THREAD

**Issue by almirfilho** [nsonnad/metalsmith-slug#7]
It would be nice the possibility to set the `slug` property manually in a single file. That way, the value of `slug` on this file may not change.

Let's say we have 3 files: `one.md`, `two.md` and `three.md`, and `three.md` has the `slug` property set manually with a different value:

__one.md:__
```
---
title: One Pig
---
Contents in One...
```

__two.md:__
```
---
title: Two Pigs
---
Contents in Two...
```

__three.md:__ (with `slug` set manually)
```
---
title: Three Pigs
slug: how-many-pigs-i-want-here
---
Contents in Three...
```

The output of processed files should be:
```javascript
{
  'one.md':   {title: 'One Pig',    slug: 'one-pig'},
  'two.md':   {title: 'Two Pigs',   slug: 'two-pigs'},
  'three.md': {title: 'Three Pigs', slug: 'how-many-pigs-i-want-here'},
}
```

What do you think @nsonnad? I can submit a PR for this feature today if you agree. =)

**Comment by nsonnad** [comment]
Good idea, will definitely take a PR. Thanks!

**Comment by nsonnad** [comment]
Fixed by #8
